src/python/competition.py

```diff
@@ -422,7 +422,7 @@ def __init__(self, cold=False,
                # Pass fixed randomSeed so separate runs are comparable (pick the same tasks):
                randomSeed=None,
                benchSearch=True,
-               taskCountPerCat = 1,
+               taskCountPerCat = 20,
```

Comment:
This is important, to trigger multiple implementations of the `IntersectVisitor` in `PointRangeQueries`

Reply:
I should also note that this might make the benchmark slower, since more implementations of the `IntersectVisitor` might drag down performance due do virtual calls now being used. 

In real Lucene applications these multiple implementations is probably the norm though, so that makes the benchmark better. https://github.com/apache/lucene/pull/13149 should lower the performance decrease from this.